### DISCUSSION TITLE
Are the valhalla tiles backwards compatible?

### DISCUSSION THREAD

**Discussion by newcarrotgames** [valhalla/valhalla#3985] [Q&A] [answered]
Can we take tiles created by an older version of Valhalla and use them with newer versions?

**Comment by nilsnolde** [comment, marked as answer]
Yes absolutely. That’s the main thing we Version in fact, apart from the JSON HTTP API.

Good reminder to update this: http://valhalla.github.io/valhalla/releasing/. Doesn’t mention that at all currently. Should go in the main README, if it’s not there already.

**Reply by nilsnolde** [reply]
To expand: all versions within a major version are compatible. Both ways: new tiles, old client and old tiles, new client. Just some features might not be available to a client.

**Reply by kevinkreiser** [reply]
yeah we have this almost properly documented in https://github.com/valhalla/valhalla/blob/master/docs/docs/releasing.md

there are a couple lies in there though

**Reply by nilsnolde** [reply]
Oops yeah wasn’t reading right, it’s actually in there.. I’ll give it a stab tmrw to amend though.
